src/System.Private.CoreLib/shared/System/IO/BinaryReader.cs

```diff
@@ -220,13 +223,10 @@ public virtual int Read()
             return _singleChar[0];
         }
 
-        public virtual bool ReadBoolean()
-        {
-            FillBuffer(1);
-            return (_buffer[0] != 0);
-        }
+        public virtual byte ReadByte() => InternalReadByte();
 
-        public virtual byte ReadByte()
+        [MethodImpl(MethodImplOptions.AggressiveInlining)]
+        private byte InternalReadByte()
         {
             // Inlined to avoid some method call overhead with FillBuffer.
```

Comment:
This comment does not make sense now since FillBuffer is not ever called anywhere else.

Reply:
fixed.